### ISSUE TITLE
Impaired renal tubular reabsorption of glucose

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6865]
**Preferred term label:**
Impaired renal tubular reabsorption of glucose

**Synonyms**

**Definition (free text, please give PubMed ID)**
Decreased renal tubular reabsorption of glucose with glucosuria in nondiabetic patients with an euglycemic status. 
PMID: 31474092, PMID: 28008953

**Parent term (use hpo.jax.org/app)**
Abnormality of renal resorption HP:0011038

**Associated term**
Renal Fanconi syndrome HP:0001994

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
Is this different from the term Glycosuria, i.e., is there a specific measurement to prove a defect in the tubule as opposed to having hyperglycemia or is this assumed from the clinical situation?

**Comment by pnrobinson** [comment]
adding with this definition:
Increased fractional excretion (FE) of glucose in the urine. The FE of glucose is the percentage of the glucose filtered by the kidney which is excreted in the urine. It is measured in terms of plasma and urine glucose, rather than by the interpretation of urinary glucose concentration alone, as urinary glucose concentrations can vary with water reabsorption.
